redisinsight/api/provider-interfaces/models/database.ts

```diff
@@ -0,0 +1,61 @@
+import { CaCertificate } from './ca-certificate';
+import { ClientCertificate } from './client-certificate';
+
+export class Database {
```

Comment:
In SaaS there will be no listing of databases; just a single database per session, with the values retrieved from the SM using DAS (Data Access Service). 
This means the provider will only implement `get` operations